sdks/go/pkg/beam/core/state/state_test.go

```diff
@@ -116,6 +117,50 @@ func (s *fakeProvider) ExtractOutputFn(userStateID string) reflectx.Func {
 	return nil
 }
 
+// ReadMapStateValue(userStateID string, key interface{}) (interface{}, []Transaction, error)
```

Comment:
Did you mean to add doc comments for the functions below?

Reply:
Whoops, no this is just a leftover bad copy paste as I moved the definitions over. I shouldn't need doc comments for test functions, but I did remove these. Good catch!